test/index.test.ts

```diff
@@ -1,31 +1,50 @@
 import { strictEqual, deepStrictEqual } from "assert";
 import { describe, test } from "mocha";
-import { getAccounts, getConnection } from "@tableland/local";
+import { getAccounts, getDatabase, getValidator } from "@tableland/local";
 
 describe("index", function () {
   // Note that we're using the second account here
   const [, signer] = getAccounts();
-  const sdk = getConnection(signer);
+  const db = getDatabase(signer);
+  const validator = getValidator();
 
   test("create", async function () {
-    const { name } = await sdk.create("counter integer", { prefix: "table" });
-    strictEqual(name, "table_31337_2");
+    const { meta } = await db
+      .prepare("CREATE TABLE my_table (counter integer);")
+      .all();
+    strictEqual(meta.txn?.name, "my_table_31337_2");
   });
 
   test("insert", async function () {
-    const { hash } = await sdk.write("insert into table_31337_2 values (1);");
-    const txnReceipt = await sdk.receipt(hash);
+    const { meta } = await db
+      .prepare("insert into my_table_31337_2 values (1);")
+      .all();
```

Comment:
While rewriting this test to work with the new major versions of tableland I found what might be a pain point, and an opportunity to come up with a best practice.
see inline comments below...